api/src/main/java/org/eclipse/microprofile/lra/annotation/Compensate.java

```diff
@@ -53,8 +53,8 @@
  *     </code>
  * </pre>
  * would be a valid compensation method declaration. If an invalid signature is detected 
- * the {@link org.eclipse.microprofile.lra.participant.InvalidLRAParticipantDefinitionException} 
- * will be thrown during the application startup.
+ * the implementation of this specification MUST prohibit successful startup of this application
```

Comment:
successful startup of **this** application -> successful startup of **the** application